ngraph/core/include/ngraph/op/prior_box_clustered.hpp

```diff
@@ -24,6 +24,7 @@ namespace ngraph
             bool clip = true;
             float step_widths = 0.0f;
             float step_heights = 0.0f;
+            float step = 0.0f;
```

Comment:
Missing `step` description.

Reply:
We don't have the `step` attribute in the specification. Should we update the spec?

Reply:
I guess we have 3 similar attrubutes:` step_h, step_w and step`
![image](https://user-images.githubusercontent.com/21181928/125036111-5117d080-e09b-11eb-922e-abe0849627a6.png)

Reply:
Yes my mistake.

Reply:
Added the description 👍